examples/image_reconstruction.ipynb

```diff
@@ -0,0 +1,705 @@
+      "text/plain": [
+       "<Figure size 720x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light",
+      "tags": []
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "inputs = next(iter(trn_dataloader))[0]\n",
+    "input_grid = make_grid(inputs)\n",
+    "\n",
+    "fig = plt.figure(figsize=(10, 10))\n",
+    "inp = input_grid.numpy().transpose((1, 2, 0))\n",
+    "plt.imshow(inp)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "LW8LLjj0w_P4"
+   },
+   "source": [
+    "# Reconstructed images after 4 epochs of training"
```

Comment:
```suggestion
    "## Reconstructed images after 4 epochs of training"
```

Reply:
Also, this section could have a piece of text and link to the previous section.

Reply:
I like the idea of opening it after N epochs.